uni/android/build.gradle

```diff
@@ -1,12 +1,14 @@
 buildscript {
     ext.kotlin_version = '1.7.21'
+    ext.android_plugin_version = '7.2.0'
```

Comment:
Downgrading the AGP a bit since `flutter_widget_from_html` has problems with newer versions

Reply:
Maybe also open an issue then with flutter_widget_from_html to solve that problem with the AGP

Reply:
https://github.com/daohoangson/flutter_widget_from_html/issues/968